muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/organization/body/application/users/usertabs.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {connect, Dispatch} from 'react-redux';
+import {i18nType} from '~/reducers/base/i18n';
+import {MobileOnlyTabs} from "~/components/general/tabs";
+import ApplicationSubPanel from "~/components/general/application-sub-panel";
+
+interface OrganizationUserTabsProps {
+  i18n: i18nType,
+  users: any
+}
+
+interface OrganizationUserTabsState {
+  activeTab: "ACTIVE" | "INACTIVE"
+  
+}
+
+export default class OrganizationUserTabs extends React.Component<OrganizationUserTabsProps, OrganizationUserTabsState>{
+  constructor(props: OrganizationUserTabsProps){
+    super(props);
+    this.state = {
+      activeTab: "ACTIVE",
+    }
+    this.onTabChange = this.onTabChange.bind(this);
+  }
+  
+  onTabChange() {
+    alert("TAB!");
```

Comment:
TAB!

Reply:
The component wants a function to work, so I gave it this. I just tried to evade from removing things an putting them back after PR. But it's not a huge problem. Removed the component.